recipes/sregistry/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set name = "sregistry" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3092be0059ae837be1cc69e4943610e729eb3facf49f1fed05a37a75349adb48
+
+build:
+  number: 0
+  entry_points:
+    - sregistry=sregistry.client:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [win]
```

I would suggest making this `noarch` even if it doesn't really make sense on Windows:
```suggestion
  noarch: python
```
(For noarch packages only Linux is tested in the feedstock so failures won't affect anything)

---

It really makes no sense at all to do this on Windows: singularity is not available for that platform. Also it would mean [patching](https://github.com/conda-forge/staged-recipes/pull/8441#issuecomment-495237500) the source code, which IMHO is silly.

---

There is no need to patch it as I'm happy to merge this even if Windows fails. The benefit is that this is built as 1 package instead of 6.

---

Ok then...